custom_components/moonraker/__init__.py

```diff
@@ -146,12 +146,22 @@ async def async_get_cameras(self):
         """Return list of cameras"""
         return await self._async_fetch_data("server.webcams.list", None)
 
+    async def async_get_printer_object_list(self):
+        """Return printer objects"""
+        return await self._async_fetch_data("printer.objects.list", None)
+
     def load_all_sensor_data(self):
         """pre loading all sensor data, so we can poll the right object"""
         for sensor in SENSORS:
             for subscriptions in sensor.subscriptions:
                 self.add_query_objects(subscriptions[0], subscriptions[1])
 
+    def load_sensor_list(self, sensor_list):
+        """Load a sensor list"""
+        for sensor in sensor_list:
+            for subscriptions in sensor.subscriptions:
+                self.add_query_objects(subscriptions[0], subscriptions[1])
```

Comment:
We should just allow parameter overload on the load_all_sensor_data(self)

and do, 
```python
load_all_sensor_data(self, sensor_list=SENSORS):
    """Load a sensor list"""
    for sensor in sensor_list:
        for subscriptions in sensor.subscriptions:
            self.add_query_objects(subscriptions[0], subscriptions[1])
```

Reply:
According to Pylint this is DANGEROUS

![image](https://user-images.githubusercontent.com/4152795/224596155-8c2200e4-e4d0-40aa-b339-fc1426ad36d6.png)

So I'll implement it a little differently